packages/react-native/src/index.ts

```diff
@@ -255,6 +255,7 @@ export interface PreparePayOnchainResponse {
 }
 
 export interface PrepareReceiveRequest {
+    payerAmountSat?: number
```

Comment:
Duplicated field? Already exists 2 lines below.

Reply:
Yes, it was due to the rebase. Will fix soon